workflow/rules/report.smk

```diff
@@ -1,21 +1,19 @@
-rule igv_report:
+rule vcf_report:
     input:
-        bcf="results/merged-calls/{group}.{event}.fdr-controlled.bcf",
         ref="resources/genome.fasta",
-        ref_idx="resources/genome.fasta.fai",
-        bams=lambda w: get_group_bams(w),
-        bais=lambda w: get_group_bams(w, bai=True)
+        bams=lambda w: get_batch_bams(w),
+        bcfs=lambda w: expand("results/merged-calls/{group}.{{event}}.fdr-controlled.bcf", group=get_report_batch(w))
     output:
-        report("results/igv-report/{group}.{event}.html", caption="../report/calls.rst", category="Variant calls")
-    log:
-        "logs/igv-report/{group}.{event}.log"
+        report(directory("results/vcf-report/{batch}.{event}/"), htmlindex="index.html", caption="../report/calls.rst", category="Variant calls")
     params:
-        os.path.join(os.path.dirname(workflow.snakefile), "resources/igv-report-template.html")
+        bcfs=lambda w: expand("{group}=results/merged-calls/{group}.{event}.fdr-controlled.bcf", group=get_report_batch(w), event=w.event),
+        bams=lambda w: get_batch_bams(w, True),
+        format_field = "-f DP AF OBS",
+        info_field = "-i PROB_FFPE_ARTIFACT PROB_PRESENT PROB_ARTIFACT PROB_ABSENT",
+        template = "-j workflow/resources/custom-table-report.js"
```

Comment:
You need to compose the file path relative to __file__, otherwise it is interpreted as relative to the working directory, which can be somewhere else than the sources (e.g. by using the --directory flag).